tests/test_get_excluded_volume.py

```diff
@@ -1,11 +1,18 @@
-import os,sys,glob
-import unittest  
 import pandas as pd
-from io import StringIO, BytesIO
-sys.path.insert(0, "../master/pyext/src/")
-from validation import get_input_information,utility
-from validation.excludedvolume import get_excluded_volume
+import glob
+import os
+import sys
+import unittest
 import warnings
+import tempfile
+import ihm
+
+path = os.path.abspath("master/pyext/src/")
```

Comment:
You probably want to make this path relative to that of the test script rather than the current directory, in case the script is run from a different directory. Perhaps something like `os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'master', 'pyext', 'src'))`